unitTests/ui/ResultListTest.ts

```diff
@@ -801,7 +838,7 @@ export function ResultListTest() {
             test.cmp.displayMoreResults(50).then(() => {
               Simulate.query(test.env);
               setTimeout(() => {
-                expect(infiniteScrollContainer.scrollTop).toBe(0);
+                expect(scrollToSpy).toHaveBeenCalledWith(0, window.pageYOffset + test.cmp.element.getBoundingClientRect().top);
```

Comment:
The name of the test does not match the test content at all.

If this is truly because the test was wrong/incorrect, then please also adapt the test name/description so that it match.